src/components/Pressable/GenericPressable/index.js

```diff
@@ -0,0 +1,24 @@
+import React, {forwardRef} from 'react';
+import GenericPressable from './BaseGenericPressable';
+import GenericPressablePropTypes from './PropTypes';
+
+const WebGenericPressable = forwardRef((props, ref) => (
+    <GenericPressable
+        // eslint-disable-next-line react/jsx-props-no-spreading
+        {...props}
+        ref={ref}
+
+        // change all props with accessibility* to aria-*
```

Comment:
Should this comment move down to above line 15, since those are the aria ones?

Reply:
I've change this comment to be more suitable to what is going on here